frontend/config/routes.rb

```diff
@@ -3,7 +3,7 @@
   get '/locale/set', to: 'locale#set'
 
   # non-restful checkout stuff
-  patch '/checkout/update/:state', to: 'checkout#update', as: :update_checkout
+  match '/checkout/update/:state', to: 'checkout#update', as: :update_checkout, via: [:put, :patch]
```

Comment:
@kimquy - Here, If it is required to change the checkout edit form, the method value for the form would be defined explicitly as patch only as route for put doesn't exist.